src/rules/max-line-length/index.js

```diff
@@ -50,7 +50,8 @@ export default function (maxLength, options) {
     }
 
     function checkNewline(match) {
-      let nextNewlineIndex = rootString.indexOf("\n", match.endIndex)
+      const endOfLineIndicator = (rootString.indexOf("\r\n", match.endIndex) !== -1) ? "\r\n" : "\n"
```

Comment:
Are we 100% confident that newline characters will be consistent across the stylesheet? What if there is a `\r\n` in one place (or even in a comment) and a `\n` in another? Then we'll have some problems, right?